manager-rs/dauth-service/src/database/user_infos.rs

```diff
@@ -8,10 +8,12 @@ use crate::data::error::DauthError;
 pub async fn init_table(pool: &SqlitePool) -> Result<(), DauthError> {
     sqlx::query(
         "CREATE TABLE IF NOT EXISTS user_info_table (
-            user_info_id TEST PRIMARY KEY,
+            user_info_id TEXT NOT NULL,
             user_info_k BLOB NOT NULL,
             user_info_opc BLOB NOT NULL,
-            user_info_sqn_max BLOB NOT NULL
+            user_info_sqn_max INT NOT NULL,
+            user_info_sqn_slice INT NOT NULL,
+            PRIMARY KEY (user_info_id, user_info_sqn_slice)
```

Comment:
Yes, I was considering this. I figured this would work for the time being, and it would be easier/quicker than creating another new table. I'll add this to a list of optimizations for the future, but my focus for the moment is trying to get dAuth working first